content/en/users/im/dashboard/_index.md

```diff
@@ -0,0 +1,179 @@
+user only has to select one of the VOs he is member and one of sites that
+supports that VO. These dropdown fields are generated using the information
+available from the sites and the list of VOs the user is member.
+
+![Edit/Add a Crecential](dash_edit_cred.png)
+
+### TOSCA Templates
+
+The list of available TOSCA templates enable the user to select the required
+topology to deploy.Each TOSCA template can be labelled by the TOSCA developer
+with any "tag" that will show a ribbon displayed on the right bottom corner.
+An special "tag" is the elastic one that are used to mark the templates that
+are configured to automatically manage the elasticity of the deployed cluster.
+
+The user have to click on the "Configure" button to set the input values of the
+TOSCA template and also to select the VO, Site and Image to deploy the
+infrastructure.
+
+Initially the user can set a name to describe the infrastructure to be deployed.
+It will make easier to list infrastructures. In the firsts tabs the user can
+introduce the set of input values of the toplogy. By default there is only one
+tab called "Input Values", but the TOSCA developer can add/rename them to make
+easier the input values selection.
+
+![TOSCA input values.](dash_inputs.png)
+
+The final tab will be the "Cloud Provider Selection".In this tab the user has to
+select: first, one of the Cloud providers that has been previously added (and
+not disabled) in the "Cloud Crecentials" page, then it has to select the base
+image used to deploy the VMs.In case of EGI Cloud Compute sites the user has two
+options, he can select an image from the list of images provided by the
+[EGI AppDB information system](https://appdb.egi.eu/) or from the list provided
+directly by the Cloud site.
+
+Other providers will only show a dropdown list with the available images to use.
+Only in the case of AWS Cloud provider the user has to specify manually the AMI
+ID of the image to use.
+
+![Select Cloud Provider and Image](dash_site.png)
+
+### Infrastructures
+
+This page will show the list of infrastructures deployed by the current user.
+The first column shows the name set by the user on infrastructure creation,
+then shows the ID assinged by the IM service, third column shows the current
+status of the infrastructure, fourth show the list of VMs with their IDs and
+finally appears a button with a set of actions to perform to it.
+
+![List of infrastructures.](dash_inf_list.png)
+
+#### List of Actions
+
+The following figure shows the list of actions available to manage the existing
+infrastructures:
+
+![List of infrastructure Actions](dash_inf_actions.png)
+
+- Add nodes: The Add nodes action enables to add new VMs to the users'
+  deployment. As depicted in it will show the list of different types of nodes
+  currently deployed in the infrastructure and the user have to set the number
+  of nodes of each type he wants to deploy.
```

Comment:
```suggestion
  deployment. It will show the list of the different types of nodes
  currently deployed in the infrastructure and the user must set the number
  of nodes of each type he wants to deploy.
```